csharp/dotnet/security/audit/mass-assignment.yaml

```diff
@@ -1,26 +1,31 @@
 rules:
 - id: mass-assignment
-  patterns:
-  - pattern-either:
-    - pattern: |
-        public IActionResult $METHOD(...){
+  mode: taint
+  pattern-sources:
+    - patterns:
+      - focus-metavariable: $ARG
```

Comment:
For readability I prefer `focus-metavariable` at the end. In my mind it works like this: after a match has been found with one of the other patterns, the focus is placed what was matched for this metavariable.

Reply:
fixed!